spec/rails_helper.rb

```diff
@@ -14,6 +14,9 @@
 
 require 'rspec/rails'
 
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.